Pod/Classes/TableViewHelpers.swift

```diff
@@ -30,23 +30,23 @@ extension UITableView {
 }
 
 public enum ReusableCell<Cell> {
-    case Class(identifier: String)
-    case Nib(identifier: String, nibName: String, bundle: NSBundle?)
+    case `class`(identifier: String)
```

Comment:
>
“To use a reserved word as an identifier, put a backpack (\`) before and after it. For example, class is not a valid identifier, but \`class\` is valid. The backticks are not considered part of the identifier; \`x\` and x have the same meaning.”

Excerpt From: Apple Inc. “The Swift Programming Language (Swift 2.2).” iBooks. https://itun.es/us/jEUH0.l